lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -337,16 +340,20 @@ def test_masquerade_as_specific_user_on_self_paced(self):
         content = response.content
         self.assertIn("OOGIE BLOOGIE", content)
 
+    @ddt.data('john', 'foo@bar')
```

Comment:
Worth mentioning the `@` and `ENABLE_UNICODE_USERNAME` here instead of the other file.